src/js/controller/instream-adapter.js

```diff
@@ -330,13 +321,13 @@ var InstreamAdapter = function(_controller, _model, _view, _mediaPool) {
 
         // Re-attach the controller & resume playback
         // when instream was inited and the player was not destroyed\
-        _controller.attachMedia(_oldpos);
+        _controller.attachMedia();
 
         if (this.noResume) {
             return;
         }
 
-        if (_oldpos === null) {
+        if (_controller.isBeforeComplete() || _model.get('state') === STATE_COMPLETE) {
```

Comment:
Have you tested postrolls? Does isBeforeComplete remain true until the postroll is complete in IMA and VAST, or is it only true when the break starts?